crates/biome_rule_options/src/no_unknown_attribute.rs

```diff
@@ -0,0 +1,10 @@
+use biome_deserialize_macros::Deserializable;
+use serde::{Deserialize, Serialize};
+#[derive(Default, Clone, Debug, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
+pub struct NoUnknownAttributeOptions {
+    pub require_data_lowercase: bool,
```

Comment:
The `require_data_lowercase` option is still here, please remove it as discussed here: https://github.com/biomejs/biome/pull/7774#discussion_r2451402749